requirements-dev.txt

```
@@ -0,0 +1,3 @@
+requests==2.19.0
```

```suggestion
-r requirements.txt
```
Let's replace this line with this instead. This way when you update the main depenencies, the dev ones update automatically.